### PR TITLE
Added reports service config to map db service name to display name on reports

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-638

### Change description ###
Added new config to map db service names to display names on report.
Updated reports service to use the new config.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sendletter/model/out/LettersCountSummary.java [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 public class LettersCountSummary {
 
-    public final String service;
+    public final String serviceName;
```
Comment (aliveni) [inline src/main/java/uk/gov/hmcts/reform/sendletter/model/out/LettersCountSummary.java]: renamed property `service` to `serviceName` to match with the config

File: src/main/java/uk/gov/hmcts/reform/sendletter/config/ReportsServiceConfig.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.Map;
+
+@ConfigurationProperties(prefix = "reports")
+public class ReportsServiceConfig {
+
+    private Map<String, String> serviceConfig;
+
+    public static class Mapping {
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/sendletter/config/ReportsServiceConfig.java]: is this class ever used?

File: src/main/java/uk/gov/hmcts/reform/sendletter/services/ZeroRowFiller.java [hunk 3]
```diff
@@ -33,8 +35,8 @@ public ZeroRowFiller(ServiceFolderMapping serviceFolderMapping) {
 
     private Set<String> missingServiceFolders(List<LettersCountSummary> listToFill) {
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/sendletter/services/ZeroRowFiller.java]: no longer folders

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/sendletter/services/ZeroRowFiller.java]: I'll rename it in the next pr

**Pull request merged**